golden_response/staging/get_stat_ranking/california.json

```diff
@@ -442,12 +442,12 @@
           {
             "rank": 29,
             "value": 37.9,
-            "placeDcid": "geoId/06095"
+            "placeDcid": "geoId/06111"
```

Comment:
Hmm... when values match, should sort by place to ensure deterministic diffs. I'll add a note to fix this.